packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/helpers/fetchRecentSubjects/fetchTalkData.js

```diff
@@ -0,0 +1,13 @@
+import talkClient from 'panoptes-client/lib/talk-client'
+
+async function fetchTalkData (projectId) {
+  return talkClient.type('comments')
```

Comment:
I think you can use panoptes.js instead even for other hosts, so we don't have to rely on the old Talk client:

```
const query =  {
      focus_type: 'Subject',
      page_size: 10,
      section: `project-${projectId}`,
      sort: '-created_at'
  }

return panoptes.get('/comment', query, {}, 'https://talk.zooniverse.org')
```
the empty object parameter is for any headers you might need to set, like authorization.

Reply:
Yup. The host value can be configured as an env var or kept in a store (or the former used to populate the latter). A similar store was rejected in #499. Do you have a preference?

Reply:
I _think_ this is the only place in the new project app that'll be making a request to the Talk API. I would prefer the least bit of code that doesn't rely on the old javascript client, so env vars and using the panoptes.js get request? If we find that we'll be making more requests to Talk in the future, then we can decide on how we'll handle it (maybe a talk.js client?).

Reply:
Sure. I'm going to argue against baking in more config again if we get to that point, but that's a discussion for another day :)

Reply:
Done